index.html

```diff
@@ -25,6 +22,24 @@ <h1 id="logo">Aziz Mejri</h1>
           </li>
         </ul>
       </nav>
+      <nav class="navbar">
+        <h1>
+          <a href="#intro" class="navbar-brand" id="logo">Elijah Ayandokun</a>
+        </h1>
+        <ul>
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href="#contact"
+              >Contact</i
+            ></a>
+          </li>
+        </ul>
+        <button type="button" class="navbar-toggler">
+          <i class="fas fa-bars"></i>
+          <i class="fa fa-times"></i>
+        </button>
```

Comment:
- [ ] _[Required]_ As shown in the Figma design you've chosen, the `X` sign/button should appear once the drop-down mobile menu shows so you can close the menu. In other words, when the user clicks (or taps) the close(X) button, the mobile menu closes.